lib/actions/actions.js

```diff
@@ -35,12 +35,14 @@ actions._prepCopy = function _prepCopy(source, destination, process) {
   destination = this.isPathAbsolute(destination) ? destination : path.join(this.destinationRoot(), destination);
 
   var encoding = null;
-  var binary = isBinaryFile(source);
-  if (!binary) {
-    encoding = 'utf8';
-  }
+  var binary = true;
+
+  body = fs.readFileSync(source);
 
-  body = fs.readFileSync(source, encoding);
+  if (istextorbinary.isTextSync(undefined, body)) {
```

Comment:
It changes `var binary` value. You should get rid of this variable.

It is easier to think about a thing that doesn't change than mutable variable values.